docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,179 @@
+ms.date: 11/27/2018
+---
+
+
+# Work with workbooks using the Excel JavaScript API
+
+This article provides code samples that show how to perform common tasks with workbooks using the Excel JavaScript API. For the complete list of properties and methods that the **Workbook** object supports, see [Workbook Object (JavaScript API for Excel)](/javascript/api/excel/excel.workbook). This article also covers workbook-level actions performed through the [Application](/javascript/api/excel/excel.application) object.
+
+The Workbook object is the entry point for your add-in to interact with Excel. It maintains collections of worksheets, tables, PivotTables, and more, through which Excel data is accessed and changed. The [WorksheetCollection](/javascript/api/excel/excel.worksheetcollection) object gives your add-in access to all the workbook's data through indivual worksheets. Specifically, it lets your add-in add worksheets, navigate among them, and assign handlers to worksheet events. The article [Work with worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md) describes how to access and edit worksheets.
+
+## Get the active cell or selected range
+
+The Workbook object contains two methods that get a range of cells the user or add-in has selected: `getActiveCell()` and `getSelectedRange()`. `getActiveCell()` gets the active cell from the workbook as a [Range object](/javascript/api/excel/excel.range). The following example shows a call to `getActiveCell()`, followed by the cell's address being printed to the console.
+
+```js
+Excel.run(function (context) {
+    var activeCell = context.workbook.getActiveCell();
+	activeCell.load("address");
+
+    return context.sync().then(function () {
+		console.log("The active cell is " + activeCell.address);
+	});
+}).catch(errorHandlerFunction);
+```
+
+The `getSelectedRange()` method returns the currently selected single range. If multiple ranges are selected, an InvalidSelection error is thrown. The following example shows a call to `getSelectedRange()` that then sets the range's fill color to yellow.
+
+```js
+Excel.run(function(context) {
+	var range = context.workbook.getSelectedRange();
+	range.format.fill.color = "yellow";
+	return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+## Create a workbook
+
+Your add-in can create a new workbook, separate from the Excel instance in which the add-in is currently running. The Excel object has the `createWorkbook` method for this purpose. When this method is called, the new workbook is immediately opened and displayed in a new instance of Excel. Your add-in remains open and running with the previous workbook.
+
+```js
+Excel.createWorkbook();
+```
+
+The `createWorkbook` method can also create a copy of an existing workbook. The method accepts a base64-encoded string representation of an .xlsx file as an optional parameter. The resulting workbook will be a copy of that file, assuming the string argument is a valid .xlsx file, 
```

Comment:
At the end of the final sentence, replace the comma with a period.

Reply:
Hmm, that's a very subjective opinion, but I'll adjust my style for you this time.